src/libs/colorpicker.c

```diff
@@ -251,9 +295,35 @@ static gboolean _sample_tooltip_callback(GtkWidget *widget, gint x, gint y, gboo
 
   for(int i = 0; i < 3; i++)
   {
-    const float *rgb = i == 0 ? sample->picked_color_rgb_mean :
-                       i == 1 ? sample->picked_color_rgb_min :
-                                sample->picked_color_rgb_max;
+    float rgb[3];
+    float rgb_strings[3];
+
+    if(i == 0)
```

Comment:
remove the whole condition and write:

`for(size_t c = 0; c < 3; c++) rgb[c] = rgb_strings[c] = picked_rgb[c];`

A single for loop will make the code lighter and easier to read.

Reply:
done